homeassistant/components/sensor/homematicip_cloud.py

```diff
@@ -89,38 +81,6 @@ def device_state_attributes(self):
         return {}
 
 
-class HomematicipDeviceStatus(HomematicipGenericDevice):
```

Comment:
Was this never used?

Reply:
Hi Martin, when I started this code was used but after the cleanup of all the state attributes I used this code was not used anymore.
Btw. is it OK to hide attributes e.g. low battery if the battery of the device is OK? 
`if hasattr(device, 'lowBat') and device.lowBat:
    return {  ATTR_LOW_BATTERY: self._device.lowBat }`

Reply:
So it's not a breaking change to remove this class?

Regarding attributes: As long as a template can be used to check for low battery, it's ok.

Reply:
This class can be removed, it's not used anywhere.
I will file a extra PR for the battery attributes in order not to mix too many things up.

Thanks Mattias